README.md

```diff
@@ -94,6 +94,13 @@ It's mainly a set of ideas, documentation and tools to use existing open source
 ssh root@172.173.174.175
 ```
 
+Check the Linux kernel of the VPS; it should **not** be in KVM. (the ipvs module, essential for swarm binding, is not present in KVM)
+
+```bash
+uname -r
+```
```

Comment:
Thanks! Could you please an an example output here showing what should be spotted?